python/Ganga/GPIDev/Base/Objects.py

```diff
@@ -41,7 +42,7 @@ def _getName(obj):
 
 _imported_GangaList = None
 
-do_not_copy = ['_index_cache', '_parent', '_registry', '_data', '_read_lock', '_write_lock', '_proxyObject']
+do_not_copy = ['_index_cache', '_index_cache_dict', '_parent', '_registry', '_data', '_data_dict', '_read_lock', '_write_lock', '_proxyObject']
```

Comment:
Only word of caution was copying large gangalists of diracfiles (aka the book-keeping perfomance bug in LHCb) used to show that the repr of a gangalist of 5k files was ending up in the memo object which caused a bottleneck. I don't remember what memo is supposed to be off the top of my head